Sources/UI/Camera/Camera.swift

```diff
@@ -13,22 +13,27 @@ public class Camera: NSObject {
   var device: AVCaptureDevice? {
     switch cameraPosition {
     case .back:
-      return cameraService.isCameraAvailable(position: .back) ? AVCaptureDevice.default(for: .video) : nil
+      return isCameraAvailable(position: .back) ? AVCaptureDevice.default(for: .video) : nil
     case .front:
-      return cameraService.isCameraAvailable(position: .front) ? AVCaptureDevice.default(.builtInWideAngleCamera, for: .video, position: .front) : nil
+      return isCameraAvailable(position: .front) ? AVCaptureDevice.default(deviceType, for: .video, position: .front) : nil
     }
   }
   let session = AVCaptureSession()
   // Communicate with the session and other session objects on this queue.
   let sessionQueue = DispatchQueue(label: "com.poviokit.camera")
-  public lazy var previewLayer = AVCaptureVideoPreviewLayer(session: session)
-  public let cameraService: CameraPermissionProviding
-  public var cameraPosition: CameraPosition = .back
+  public lazy var previewLayer: AVCaptureVideoPreviewLayer = {
+    let pl = AVCaptureVideoPreviewLayer(session: session)
+    pl.videoGravity = AVLayerVideoGravity.resizeAspectFill
+    
+    return pl
```

Comment:
Try to avoid using abbreviations for variable names. Naming it `previewLayer` is much clearer.